services/src/contexts/migrations/migration_0015_log_quota.rs

```diff
@@ -12,15 +9,32 @@ pub struct Migration0015LogQuota;
 #[async_trait]
 impl Migration for Migration0015LogQuota {
     fn prev_version(&self) -> Option<DatabaseVersion> {
-        Some(Migration0014MultibandColorizer.version())
+        None // must have migrated to 0015 with old version
```

Comment:
maybe add some more info...
- why we start with 0015 and not 0001 -> because we performed major refactoring when joining pro and non-pro codebase
- specify which version (commit hash?) to use to mgirate 0015

Reply:
[b697ee8](https://github.com/geo-engine/geoengine/pull/1009/commits/b697ee8a8194c602e1fb8bde36af9ba6b61b9fe1)